pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("Apply", func() {
 
 	Context("Services", func() {
 
-		It("should patch if ClusterIp == \"\" during update", func() {
+		It(`should not patch if ClusterIp == "" during update`, func() {
```

Comment:
This change (and there are other I think), do not fit the same commit (or even PR) as the subject.
Can you extract it to a different PR?